piker/brokers/binance.py

```diff
@@ -439,6 +519,35 @@ async def get_ohlc(
         yield get_ohlc, {'erlangs': 3, 'rate': 3}
 
 
+@async_lifo_cache()
+async def get_mkt_info(
```

Comment:
this is the new **required** endpoint for all backends which delivers the `MktPair` (ideally with embedded `.src`/`.dst` attrs which are `Asset`s) translation for any searched market/symbol/ticker pattern.